src/core.js

```diff
@@ -31,6 +31,7 @@ exports.moveTraces = Plotly.moveTraces;
 exports.purge = Plotly.purge;
 exports.setPlotConfig = require('./plot_api/set_plot_config');
 exports.register = Plotly.register;
+exports.toImage = require('./plot_api/to_image');
```

Comment:
the way you wrote `snapshot/download.js`, I think it would as easy as adding:

```js
exports.downloadImage = require('./snapshot/download');
```

below this line.